substrate/client/cli/src/params/node_key_params.rs

```diff
@@ -79,22 +80,47 @@ pub struct NodeKeyParams {
 	/// the chosen type.
 	#[arg(long, value_name = "FILE")]
```

Comment:
While you are at it, may be make `--node-key` & `--node-key-file` conflicting arguments?